storage/tokudb/tokudb_information_schema.cc

```diff
@@ -1119,9 +1119,9 @@ void background_job_status_callback(
     table->field[3]->store(type, strlen(type), system_charset_info);
     table->field[4]->store(params, strlen(params), system_charset_info);
     if (user_scheduled)
-        table->field[5]->store("USER", sizeof("USER"), system_charset_info);
+        table->field[5]->store("USER", strlen("USER"), system_charset_info);
```

Comment:
Yes, I already have a 5.6 branch ready to go since we aren't in GCA mode 
yet.

On 12/03/2015 07:20 AM, Laurynas Biveinis wrote:
>
> In storage/tokudb/tokudb_information_schema.cc 
> <https://github.com/laurynas-biveinis/percona-server/pull/1#discussion_r46555614>:
>
> > @@ -1119,9 +1119,9 @@ void background_job_status_callback(
> >      table->field[3]->store(type, strlen(type), system_charset_info);
> >      table->field[4]->store(params, strlen(params), system_charset_info);
> >      if (user_scheduled)
> > -        table->field[5]->store("USER", sizeof("USER"), system_charset_info);
> > +        table->field[5]->store("USER", strlen("USER"), system_charset_info);
>
> Should this go to 5.6 as well? Not right now with an upmerge or 
> anything like that, just to make a note somewhere right now
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/laurynas-biveinis/percona-server/pull/1/files#r46555614>.
>

-- 
George O. Lorch III
Software Engineer, Percona
US/Arizona (GMT -7)
skype: george.ormond.lorch.iii